acquire/acquire.py

```diff
@@ -830,6 +830,7 @@ class AV(Module):
         ("dir", "sysvol/ProgramData/McAfee/Endpoint Security/Logs_Old"),
         ("dir", "sysvol/ProgramData/Mcafee/VirusScan"),
         ("dir", "sysvol/ProgramData/McAfee/Endpoint Security/Logs"),
+        ("glob", "sysvol/ProgramData/McAfee/MSC/Logs/*.log"),
```

Comment:
Any particular reason why we wouldn't collect the entire `Logs` directory? Like the other lines do.

Reply:
Not sure what else there is in that directory, might be redundant, but okay.

Reply:
Better to collect too much than not enough :smile: